faker/providers/person/fr_FR/__init__.py

```diff
@@ -259,45 +259,997 @@ class Provider(PersonProvider):
+        'Chnadonnet',
+        'Lacroix',
+        'Allain',
+        'Jetté',
+        'Vadnais',
+        'Rouleau',
+        'Chauvet',
+        'Durand',
+        'Sarrazin',
+        'Labbé',
+        'Pelland',
+        'Ouellet',
+        'Narcisse',
+        'Lécuyer',
+        'Lavoie',
+        'Patry',
+        'Rousseau',
+        'Lapierre',
+        'Savoie',
+        'Marquis',
+        'Bourgouin',
+        'Cailot',
+        'Jodoin',
+        'Fugère',
+        'David',
+        'Racicot',
+        'Ferland',
+        'Corbeil',
+        'Gagnon',
+        'Berthiaume',
+        'Proulx',
+        'Leblanc',
+        'Sanschagrin',
+        'Chassé',
+        'Busque',
+        'Gaillard',
+        'Boivin',
+        'Nadeau',
+        'Aubé',
+        'Brousseau',
+        'LHeureux',
+        'Ricard',
+        'Pelchat',
+        'Bordeaux',
+        'Mailly',
+        'Compagnon',
+        'Sauriol',
+        'Doucet',
+        'Guernon',
+        'Leroux',
+        'Plouffe',
+        'Laforge',
+        'Bienvenue',
+        'Lacombe',
+        'Laberge',
+        'Lanteigne',
+        'LeBatelier',
+        'Therriault',
+        'Simard',
+        'Plourde',
+        'Ruest',
+        'Lepage',
+        'Goulet',
+        'Paiement',
+        'Adler',
+        'Favreau',
+        'Mireault',
+        'Sylvain',
+        'Beaulac',
+        'Guertin',
+        'Trudeau',
+        'Chartré',
+        'Dagenais',
+        'Auger',
+        'Duplessis',
+        'Laframboise',
+        'Daigle',
+        'Roy',
+        'Bérard',
+        'Gamache',
+        'Paimboeuf',
+        'Barrette',
+        'Carignan',
+        'Cyr',
+        'Dastous',
+        'Talon',
+        'Giroux',
+        'Gaillou',
+        'Cantin',
+        'Longpré',
+        'Levasseur',
+        'Cousteau',
+        'Marchesseault',
+        'Bolduc',
+        'Bellemare',
+        'Patenaude',
+        'Deserres',
+        'Guérette',
+        'Bureau',
+        'Gabriaux',
+        'Bonneville',
+        'Lamarre',
+        'Lesage',
+        'Meunier',
+        'Beaudry',
+        'Panetier',
+        'Abril',
+        'Laliberté',
+        'Grenier',
+        'Laprise',
+        'Théberge',
+        'Salois',
+        'Beaudouin',
+        'St-Jacques',
+        'Picard',
+        'Mercier',
+        'Brian',
+        'Coudert',
+        'Chouinard',
+        'Dubé',
+        'Brisette',
+        'Gousse',
+        'Lebrun',
+        'Varieur',
+        'Labelle',
+        'Duperré',
+        'Grondin',
+        'Lapointe',
+        'Massé',
+        'Chastain',
+        'Ratté',
+        'Lacharité',
+        'Charest',
+        'Royer',
+        'Paradis',
+        'Barteaux',
+        'Norbert',
+        'Parenteau',
+        'Chrétien',
+        'Sorel',
+        'Quenneville',
+        'Casgrain',
+        'Faure',
+        'Bourgeois',
+        'Laramée',
+        'Houde',
+        'Tremblay',
+        'Arpin',
+        'LAngelier',
```

Comment:
Is this a typo? should it not be `Langelier`?

Reply:
I think this can be both, either "L'Angelier" (without the single quote) or Langelier. Depends on the DataSource  I guess. (You might see the same "issue" on "LHiver", which can be written both ways)